manager/manager.go

```diff
@@ -768,11 +783,44 @@ func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *c
 	return nil
 }
 
+// Enables overwriting an existing containerData/Handler object for a given containerName.
+// can't use createContainer as it just returns if a given containerName has a handler already
+// Ex: rkt handler will want to take priority over the raw handler, but the raw handler might be created first
+
+// Only allow raw handler to be overriden
+func (m *manager) overrideContainer(containerName string, watchSource watcher.ContainerWatchSource) error {
+	m.containersLock.Lock()
+	defer m.containersLock.Unlock()
+
+	namespacedName := namespacedContainerName{
+		Name: containerName,
+	}
+
+	if _, ok := m.containers[namespacedName]; ok {
+		containerData := m.containers[namespacedName]
+
+		if containerData.handler.String() != "raw" {
+			return nil
+		}
```

Comment:
Its now removed in this branch and is part of the rkt addons branch, but the reason is to only allow overriding a raw handler.

i.e. cgroup gets a raw handler, then we say it should get a rkt handler.  it just to enforce.  If you dont think its neccessary can remove it